### PR TITLE
Added the file extension for a VS Code workspace file to .gitignore. …

### LINKED ISSUE #188: Missing converter: use-default-type-parameter
### 💥 Missing Converter

-   `tslint-to-eslint-config` version: 1.2.0
-   `typescript-eslint` version: 2.3.2

#### TSLint Rule Name

`use-default-type-parameter`

### PR DESCRIPTION
**Pull request by tonythetiger323**
…Created a test for use-default-type-parameter, add the rule to rules/converters.ts, and created it it's own converter file.

## PR Checklist

-   [ x] Addresses an existing issue: fixes #188
-   [ x] That issue was marked as [`status: accepting prs`](https://github.com/typescript-eslint/tslint-to-eslint-config/labels/status%3A%20accepting%20prs)

## Overview
It was pretty straightforward as its one of the few rule tslint rules that doesn't have an eslint equivalent which ban-ts-ignore is pretty much the same so I just followed the way it was laid out to a t. Um, tangent, there appears to be something wrong with your e2e and I briefly looked at it but don't have much e2e testing experience. I can create and issue if you'd like.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,5 @@ test/*.js
 ~test/jest.config.js
 !test/tests/**/.eslintrc*
 !test/tests/**/*.log
+#VS Code workspace file
+*.code-workspace
```
Comment (JoshuaKGoldberg) [inline .gitignore]: Err this and the PR title look unrelated. Could you please file a separate issue if you'd like this change?

File: src/rules/converters/use-default-type-parameter.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { RuleConverter } from "../converter";
+
+export const convertUseDefaultTypeParameter: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "@typescript-eslint/use-default-type-parameter",
```
Comment (JoshuaKGoldberg) [inline src/rules/converters/use-default-type-parameter.ts]: It's actually this rule now: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unnecessary-type-arguments.md

**Comment by JoshuaKGoldberg** [comment]
Just a heads up @tonythetiger323, #221 was already merged in (the author had asked early on about this rule). Are you still interested in the .gitignore changes?

File: .gitignore (hunk 1 above)
Comment (tonythetiger323) [reply .gitignore]: My apologies for the delay... I went out of town and had no service so was pretty disconnected, glad to see it got taken care of. Well, if you would like to have code-workspaces ignored I'm not opposed to resbubmitting.

**Comment by JoshuaKGoldberg** [comment]
@tonythetiger323 if you merge from master, the CI failures for the `test-end-to-end` will go away. Sorry if that was confusing!

**Comment by JoshuaKGoldberg** [comment]
Closing to keep the list of open PRs nice and tidy. Please do resubmit if you're still interested in sending this contribution in! 💖
